masterlist.yaml

```
@@ -9965,6 +9965,14 @@ plugins:
       - Invent.Add
       - Relev
 
+  - name: '(Oblivion|AllDlC''s|Armamentarium|Armory of the Silver Dragon|Immersive Weaspons|Better Cities Resources)-WeaponWeight.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/47321/'
+        name: 'More Realistic Weapon Weight'
+    tag:
+      - Filter
+      - Stats
```

```suggestion
    tag:
      - Deactivate
      - NoMerge
      - Stats
  - name: 'AllDlC''s-WeaponWeight.esp'
    tag: [ Filter ]
```

**Filter** only makes sense for the DLC plugin as all of the masters for the other plugins are always required for each. **Deactivate** & **NoMerge** would allow only the stats changes of each plugin to be merged.